### PR TITLE
PXP-664 refactor: replace global init with inquire library

### PR DESCRIPTION
**Pull request by mfauzaan**
## Summary
We are already using the new input library, inquire, for the per-application initialization flow. It would be beneficial to do the same thing with the global initialization flow as well.

https://github.com/mindvalley/wukong-cli/assets/25294487/4dcf6df2-c19e-4c65-aba4-c47a39fa428e

Ticket: https://mindvalley.atlassian.net/browse/PXP-664

### Changed 

- [x] Change the text of browser opening to make it shorter from `Your browser has been opened to visit` to `Opening browser to`
- [x] Updated loader icon to green
- [x] Updated init success message
- [x] Replace the `dialoguer`  with `inquire` 
- [x] Some color changes to meet the new styling
- [x] Removed application selector from init flow
- [x] Move the okta client id to okta group
- [x] Updated set & get config command to match new structure for okta client id

### Fixed 
- [x] Fix an issue where `Logging in to Google Cloud` does not clear after logged in

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/193#" title="Author self-approved">mfauzaan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/193#" title="Author self-approved">mfauzaan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/193#pullrequestreview-1876820749" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
